lib/school/person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
```

Comment:
* It is a requirement to create instant variable `@id` [Check here ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Instance%20vars%3A%20%40id%2C%20%40name%2C%20and%20%40age.) . Kindly fix this 🙏